test/integration/targets/ecs_ecr/tasks/main.yml

```diff
@@ -329,6 +329,39 @@
         that:
           - result is not changed
 
+
+    - name: When creating with force_delete flag true
+      ecs_ecr:
+        name: '{{ ecr_name }}'
+        region: '{{ ec2_region }}'
+        force_delete: true
+        ec2_access_key: '{{ec2_access_key}}'
```

Comment:
The documentation lists aws_access_key and I believe ec2_access_key is kinda deprecated.  At the very least it's misleading since you're not actually talking to the ec2 endpoints (you're talking to sts and ecr).

I'd recommend switching this over and making use of module_defaults.  The bock definition right at the start can be updated to:

```
- name: 'Run tests against ecs_ecr'
  module_defaults:
    group/aws:
       region: '{{ aws_region }}'
       aws_access_key: '{{ aws_access_key }}'
       aws_secret_key: '{{ aws_secret_key }}'
       security_token: '{{ security_token | default(omit) }}'
```

This then lets you delete a large number of lines of C&P :)

Reply:
Will be treated in a dedicated PR

Reply:
See #62985